src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -6542,7 +6542,6 @@ bool ClrDataAccess::GetILImageInfoFromNgenPEFile(PEAssembly *pPEAssembly,
     return true;
 }
 
-#if defined(FEATURE_CORESYSTEM)
 /* static */
 // We extract "ni.dll from the NGEN image name to obtain the IL image name.
```

Comment:
This looks like left over from Windows Phone support (Apollo = Windows Phone) and it should be deleted. We do not use .ni.dll extensions and play games between IL and NGen images in .NET Core anymore.